### PR TITLE
Feature/filter exclude state

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: valhalla/userrequests/filters.py [hunk 1]
```diff
@@ -9,6 +9,7 @@ class UserRequestFilter(filters.FilterSet):
     state = django_filters.ChoiceFilter(choices=UserRequest.STATE_CHOICES)
     title = django_filters.CharFilter(name='group_id', lookup_expr='icontains', label='Title contains')
     user = django_filters.CharFilter(name='submitter__username', lookup_expr='icontains', label='Username contains')
+    exclude_state = django_filters.CharFilter(name='state', lookup_expr='icontains', label='Exclude State', exclude=True)
```
Comment (Fingel) [inline valhalla/userrequests/filters.py]: Why don't you just negate the already existing state filter (line 9) with `exclude=True` making it a freeform field with icontains will just slow it down.

Comment (jnation3406) [inline valhalla/userrequests/filters.py]: don't we want to keep the functionality of filtering by a state as well? I am not understanding how we can have both without adding another field to filter on, but show me if there is another way.

Comment (Fingel) [inline valhalla/userrequests/filters.py]: exclude_state = django_filters.ChoiceFilter(name='state', choices=UserRequest.STATE_CHOICES, exclude=True)

Comment (jnation3406) [inline valhalla/userrequests/filters.py]: okay, in

**Pull request merged**